sources/packages/backend/apps/api/src/services/sin-validation/sin-validation.service.models.ts

```diff
@@ -1,7 +1,7 @@
 import {
   OkayFlag,
   SINCheckStatus,
-} from "../../esdc-integration/sin-validation/models/sin-validation-models";
+} from "@sims/integrations/esdc-integration/sin-validation/models/sin-validation-models";
```

Comment:
If used in multiple modules, it can go to `@sims/services`

Reply:
i think i tried, and i was getting 
![image](https://user-images.githubusercontent.com/77353155/210418314-7f286691-8eff-419b-851a-8c129e5ce290.png)

Reply:
I see the problem here.(name too generic).